packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -259,11 +259,7 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         if (this.channelDeferred) {
             this.channelDeferred.promise.then((runtime) => {
                 runtime.dispose();
-            }).catch((error) => {
-                this.logger.sendErrorEvent(
-                    { eventName: "ChannelDisposeError", fluidDataStoreId: this.id },
-                    error);
-            });
+            }).catch((error) => {});
```

Comment:
Add a comment explaining that we sent an error event in all codepaths where `channelDeferred` rejects?